### PR TITLE
Register

### PR DESCRIPTION
**Pull request by JBigalke**
Register for Collections.

### CODE REVIEW

File: my/XRX/src/mom/app/collection/widget/my-collection-mockup.widget.xml [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+       		</div>
+        </div>      
+            <script type="text/javascript">jQuery(document).ready(function(){{ 
+ $("#searchbar").collectionIndexSearch({{ requestRoot:"{conf:param('request-root')}", collectionId:"{$collectionId}", termsearch:'{session:get-attribute("_collectionSearchAsJson")}' }}); }});</script>
+   </div>
+
+       		      		
+ </div>
+     <div id="searchresults">
+    			{ if(empty(session:get-attribute("_collectionSearchCharters"))) then <div>leeer</div>
```
Comment (StephanMa) [inline my/XRX/src/mom/app/collection/widget/my-collection-mockup.widget.xml]: typo... should be `<div>leer</div>`.
is this a debug output? why no i18n

File: my/XRX/src/mom/app/collection/collectionsearch.xqm [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+
+
+import module namespace xrx="http://www.monasterium.net/NS/xrx" at "xmldb:exist:///db/www/core/app/xrx/xrx.xqm";
+import module namespace cei="http://www.monasterium.net/NS/cei" at "xmldb:exist:///db/XRX.live/mom/app/metadata/cei.xqm";
+import module namespace atom="http://www.w3.org/2005/Atom" at "xmldb:exist:///db/XRX.live/mom/app/data/atom.xqm";
+import module namespace conf="http://www.monasterium.net/NS/conf" at "xmldb:exist:///db/XRX.live/mom/app/xrx/conf.xqm";
+import module namespace jsonx="http://www.monasterium.net/NS/jsonx" at "xmldb:exist:///db/www/core/app/json/jsonx.xqm";
+
+
+
```
Comment (StephanMa) [inline my/XRX/src/mom/app/collection/collectionsearch.xqm]: Please Please Please add comments!

Comment (JBigalke) [reply my/XRX/src/mom/app/collection/collectionsearch.xqm]: I added some comments. I hope they are helpfull

Comment (icarus-eu) [reply my/XRX/src/mom/app/collection/collectionsearch.xqm]: Would it be possible to make the look of the collections-area a bit more similar to the general mom-design and nice? The current version looks rather crude:

----

annotation

Comment (icarus-eu) [reply my/XRX/src/mom/app/collection/collectionsearch.xqm]: "Browse in" results in an error message: `Expected cardinality: exactly one, got 0. [at line 161, column 26, source: String]
In function:
	metadata:uri-tokens(xs:string) [1:426:/db/XRX.live/mom/app/metadata/metadata.xqm]`

**Pull request merged**